tfac/predict.py

```diff
@@ -25,11 +25,13 @@ def run_model(data, labels):
         random_state=42
     )
 
+    # TODO: Remove validation samples here since you're passing in the labels. Clearly Unknown should be left out of training.
```

Comment:
I removed `drop_validation`. This would be the place to remove Unknown labeled samples, since it will then happen for any prediction type.